### ISSUE TITLE
`.nx/` makes all-contributors plugin think info has changed, crashes pipeline

### ISSUE THREAD

**Issue by KVSRoyal** [intuit/auto#2457]
**Describe the bug**

I am unable to find any mentions of `nx` in this repo, but for some reason, a `.nx/` folder is being added when I use `yarn auto shipit -vv` on our `master` branch.

Re: [this discussion](https://github.com/nrwl/nx/discussions/20605#discussion-5930428), when working with `nx` >= 17, the `.nx/` folder has been moved from `node_modules/.cache` to the workspace root. This is causing `changedFiles` in the code below to be true, which makes the plugin think that the README and .all-contributorsrc have been updated. The plugin tries to commit these files and errors out because those files actually have not changed.

https://github.com/intuit/auto/blob/335fadb33fc1b661faa2a82a1d992b0e81036a74/plugins/all-contributors/src/index.ts#L360-L366

<details>
<summary>Here is the error shown with `-vv`</summary>

```
The following paths are ignored by one of your .gitignore files:
fastlane/README.md
hint: Use -f if you really want to add them.
hint: Turn this message off by running
hint: "git config advice.addIgnoredFile false"
fatal: pathspec '**/.all-contributorsrc' did not match any files
On branch master
Untracked files:
  (use "git add <file>..." to include in what will be committed)
    .nx/

nothing added to commit but untracked files present (use "git add" to track)
Error: Running command 'git' with args [commit, --no-verify, -m, "Update contributors [skip ci]"] failed

On branch master

Untracked files:
  (use "git add <file>..." to include in what will be committed)
    .nx/

nothing added to commit but untracked files present (use "git add" to track)
```

</details>

**To Reproduce**

I will try to reproduce this with a small repo. TBD.

**Expected behavior**

I expect `auto` to:
1. not crash my pipeline
2. gracefully handle it when a user has not meet any new contribution criteria and `shipit` is called on `master`

**Environment information:**

```txt
Will add in a few minutes
```

**Comment by ValentinH** [comment]
Yes, same issue here. Thanks for the tip regarding the `.gitignore` 🎉

**Comment by jackw** [comment]
>We don't use .nx/ in the repo this issue was discovered on, so not sure where it's coming from

The most likely candidate is that it's coming from [Lernas dependencies](https://github.com/lerna/lerna/blob/main/packages/lerna/package.json#L85).

>I was able to workaround this on our pipeline by adding .nx/ to our .gitignore.

I think this is probably the best course of action all round as that directory is considered temporary for containing local cache data for nx. Another option would be to try setting the lerna.json property [`useNx: false`](https://github.com/lerna/lerna/blob/main/packages/lerna/schemas/lerna-schema.json#L28) and see if it prevents creation of the directory.

**Comment by hipstersmoothie** [comment]
Not sure there is anything actionable here for auto?
